### PR TITLE
Jenkins refactor chart values

### PR DESCRIPTION
**Pull request by lachie83**
Most chart follow lowerCamelCase. I figured it was time we brought this chart into line for reference.

### CODE REVIEW

**Comment by lachie83** [comment]
@viglesiasce @seanknox

**Review by seanknox — APPROVED** [review]

**Comment by viglesiasce** [comment]
I'm torn on this one. While agree that some consistency among charts is ideal, I'm not sure we have anything in place that dictates camel case. My other hold up is that this is a fully API (values file) breaking change. I'd rather not do something like that in a minor release. 

Can we wait until there is a more substantial functional change before making this cosmetic one? Then we can bump to 1.x.

**Comment by prydonius** [comment]
My 2 cents are that having consistency in values.yaml files across the repo will be a lot less confusing for chart users. For example, I've seen a lot of folks assume that the `persistence.enabled` value is the same in every chart, and whilst it may or may not make sense to assume that persistence can be configured in every chart, we can at least avoid more confusion by having a convention where this value is used. Calling this `Persistence.Enabled` is just as annoying as calling this `Volume.Enabled` or something else as it requires me to learn the values that are specific to this chart.

Agree that this is a breaking change, and should be bumped to 1.0.0, however I don't think it needs to wait for other substantial changes. @viglesiasce are there any specific changes you'd like to rope in with this?

File: stable/jenkins/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: jenkins
 home: https://jenkins.io/
-version: 0.5.0
+version: 0.6.0
```
Comment (prydonius) [inline stable/jenkins/Chart.yaml:3]: Bump to 1.0.0

**Comment by lachie83** [comment]
I was trying to achieve some consistency even if not required. I also think lowerCamelCase is easier to read (but that's subjective) I've received a bunch of comments as to why this chart doesn't look like other charts that are out there. Also was trying not to bundle a cosmetic change with a functional change so that we can isolate any bugs we caused in this change.

**Comment by viglesiasce** [comment]
I'm good with just a version bump and not requiring further changes.

**Comment by prydonius** [comment]
@lachie83 want to rebase this one and bump the version? Marking as stale in the meantime

**Comment by lachie83** [comment]
I'm happy to close this. There's not enough interest currently

**Pull request closed without merging**